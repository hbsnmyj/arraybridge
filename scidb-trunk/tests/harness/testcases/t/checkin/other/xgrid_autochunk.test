--setup
create temp array T1 <v:int64>[i=0:999,100,0,j=0:199,10,0]
--igdata "store(build(<v:int64>[i=0:99,10,0,j=0:99,10,0], i*1000 + j), T1)"

--test

# Redimension with and without autochunking, and store results.
--start-igdata
store(xgrid(redimension(T1, <v:int64>[i=0:999,100,0,j=0:199,20,0]), 2, 2), XG)
store(xgrid(redimension(T1, <v:int64>[i=0:999,*,0,j=0:199,10,0]), 2, 2), XG_ac)
--stop-igdata

# Behold the interesting schemas!
show(XG)
show(XG_ac)

# The cells are exactly the same!
filter(join(XG, XG_ac), XG.v <> XG_ac.v)

--cleanup
remove(T1)
remove(XG)
remove(XG_ac)
