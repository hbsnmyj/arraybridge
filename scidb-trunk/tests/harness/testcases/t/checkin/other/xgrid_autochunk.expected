Query was executed successfully

[Query was executed successfully, ignoring data output by this query.]

[Query was executed successfully, ignoring data output by this query.]

[Query was executed successfully, ignoring data output by this query.]

{i} schema
{0} 'XG<v:int64> [i=0:1999,200,0,j=0:399,40,0]'

{i} schema
{0} 'XG_ac<v:int64> [i=0:1999,256,0,j=0:399,20,0]'

{i,j} v,v

Query was executed successfully

Query was executed successfully

Query was executed successfully

